src/components/List/List.js

```diff
@@ -1,35 +1,97 @@
 import React, { useEffect, useState } from 'react';
+import {
+  collection,
+  doc,
+  onSnapshot,
+  serverTimestamp,
+  updateDoc,
+} from 'firebase/firestore';
 import { db } from '../../lib/firebase';
-import { collection, onSnapshot } from 'firebase/firestore';
 import Navigation from '../Navigation/Navigation';
-import { useParams } from 'react-router-dom';
+import { useParams, Link } from 'react-router-dom';
+
+export const timeDifference = (timestampInMilli) => {
+  if (!timestampInMilli) return false;
+
+  const dayInMilli = 86400000;
+  const date = Date.now();
+  const difference = date - timestampInMilli;
+  const comparison = difference < dayInMilli;
+
```

Comment:
You can merge this in as is but at some point you can remove these console.log's so as not to clutter our users' consoles.